internal/database/mysql_test.go

```diff
@@ -293,8 +304,8 @@ func TestGetUser(t *testing.T) {
 					if user == nil {
 						t.Fatal("Expected to find user, but got nil")
 					}
-					if user.Username != tc.username {
-						t.Errorf("Expected username to be '%s', but got %s", tc.username, user.Username)
+					if user.Firstname != tc.firstname {
+						t.Errorf("Expected username to be '%s', but got %s", tc.firstname, user.Firstname)
 					}
```

Comment:
Please use the pattern to check for all the fields. (middlename, lastname, etc.)